### PR TITLE
Prl 1205 ca da manage order c49

### PR DESCRIPTION
**Pull request by khurshedk**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

PRL-1205

### Change description ###

Added C49 template changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum CaseTransferOptionsEnum {
+    @JsonProperty("theHighCourt")
+    theHighCourt("theHighCourt", "The high court"),
+    @JsonProperty("countyCourt")
+    countyCourt("countyCourt", "County court"),
+    @JsonProperty("familyProceedingCourt")
+    familyProceedingCourt("familyProceedingCourt", "Family proceeding court");
```
Comment (cshiwarale) [inline src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java:18]: As per figma, text should be 'Family proceedings court'.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java [hunk 2]
```diff
@@ -610,10 +608,12 @@ public static String[] temporaryFields() {
     private final String recitalsOrPreamble;
     private final String orderDirections;
     private final String furtherDirectionsIfRequired;
-    private final List<String> cafcassEmailAddress;
-    private final List<String> otherEmailAddress;
+    //private final List<String> cafcassEmailAddress;
+    //private final List<String> otherEmailAddress;
```
Comment (cshiwarale) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java:615]: Please remove commented code

File: src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java (hunk 1 above)
Comment (khurshedk) [reply src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java]: Corrected as per figma

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java (hunk 2 above)
Comment (khurshedk) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java]: removed commented code

**Comment by khurshedk** [comment]
Closing this and creating a new PR

**Pull request closed without merging**